utils/luci-app-ffwizard-berlin/root/usr/share/rpcd/acl.d/luci-app-ffwizard-berlin.json

```diff
@@ -0,0 +1,80 @@
+{
+        "luci-app-ffwizard-berlin": {
+                "description": "Provides access to config files and scripts",
+                "read": {
+                        "cgi-io": [
+				"exec"
+			],
+                        "uci": [
+                                "profile_*",
+                                "system",
+                                "ffwizard",
+                                "dhcp",
+                                "olsrd",
+                                "olsrd6",
+                                "firewall",
+                                "freifunk",
+                                "wireless",
+                                "network",
+                                "qos",
+                                "luci_statistics",
+                                "openvpn"
+
+                        ],
+                        "files" : {
+                                "/etc/config/ffwizard": [
+                                        "read"
+                                ],
+                                "/etc/luci-uploads/cbid.ffuplink.1.cert": [
+                                        "read"
+                                ],
+                                "/etc/openvpn/ffuplink.crt": [
+                                        "read"
+                                ],
+                                "/etc/luci-uploads/cbid.ffuplink.1.key": [
+                                        "read"
+                                ],
+                                "/etc/openvpn/ffuplink.key": [
+                                        "read"
+                                ],
+                                "/usr/libexec/rpcd/ffwizard-berlin": [
+                                        "exec"
+                                ]
+                        }
+                },
+                "write": {
+                        "uci": [
+                                "profile_*",
+                                "system",
+                                "ffwizard",
+                                "dhcp",
+                                "olsrd",
+                                "olsrd6",
+                                "firewall",
+                                "freifunk",
+                                "wireless",
+                                "network",
+                                "qos",
+                                "luci_statistics",
+                                "openvpn"
+                        ],
+                        "files" : {
+                                "/etc/config/ffwizard": [
+                                        "write"
```

Comment:
Will giving access via UCI-key no be enough? Why explicitly this file?

Reply:
Maybe you are right. I was scared that I won't grant enough permissions. Maybe I should remove this again.